src/components/FeaturePanel/renderers/OpeningHoursRenderer.tsx

```diff
@@ -42,7 +43,16 @@ const formatDescription = (isOpen: boolean, days: SimpleOpeningHoursTable) => {
 
 const OpeningHoursRenderer = ({ v }) => {
   const [isExpanded, toggle] = useToggleState(false);
-  const { daysTable, isOpen } = parseOpeningHours(v);
+
+  const { countryCode, center } = useFeatureContext().feature;
+
+  const openingHours = parseOpeningHours(v, center[0], center[0], {
```

Comment:
There is probably a typo `center[0], center[0]`, right?

To prevent mistakes like these, do you think you can change the signature of `parseOpeningHours()` to accept `LonLat` type instead of two parameters?